doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -140,6 +137,8 @@ Security
 
   * Support for AES in counter mode using CRACEN for the :zephyr:board:`nrf54lm20dk`.
 
+  * Experimental support for TF-M on the nRF54LV10A SoC.
+
```

Comment:
*Experimental support for NSIB and MCUBoot on the nRF54LV10A SoC. 
*Experimental support for KMU on the nRF54LV10A SoC.
*Experimental support for compression and encryption on the nRF54LV10A SoC.